analysis/study_definition_measures.py

```diff
@@ -0,0 +1,141 @@
+from cohortextractor import Measure, StudyDefinition, patients, codelist, codelist_from_csv  # NOQA
+import pandas as pd
+
+antibacterial_codes = codelist_from_csv(
+    "codelists/opensafely-antibacterials.csv", system="snomed", column="dmd_id",
+)
+
+copd_codes = codelist_from_csv(
+    "codelists/opensafely-current-copd.csv", system="ctv3", column="CTV3ID"
+)
+
+acne_codes = codelist_from_csv(
+    "codelists-local/alexorlek-acne-59444b72.csv", system="snomed", column="code"
+)
```

Comment:
It's not necessary, but you could put these in a separate python file (example [here](https://github.com/opensafely/renal-short-data-report/blob/main/analysis/codelists.py)) and import the codelists from that file. It helps keep things neater if you have lots of codelists.